webhook/chaos_handler.go

```diff
@@ -58,6 +60,28 @@ func (m *ChaosHandlerMutator) Handle(ctx context.Context, req admission.Request)
 		podName = pod.ObjectMeta.GenerateName
 	}
 
+	handlerTimeout := m.Timeout.String()
+	succeedOnTimeout := ""
+
+	timeoutLabel, ok := pod.Annotations["chaos.datadoghq.com/disrupt-on-init-timeout"]
+	if ok {
+		if timeoutOverride, err := time.ParseDuration(timeoutLabel); err == nil {
```

Comment:
I was thinking that we don't want to reject the pod's admission if this is invalid, and just use the timeout default, but then that does make it hard to inform the user.... i'll make the change

Reply:
@aymericDD I tested, and the rejected admission never makes it to the user, and then the pod invisibly fails to get scheduled. I think it will be _much_ easier for everyone involved to debug if the pod starts with a default timeout, instead of simply never appearing

Reply:
I chose to log the warning so we have some internal visibility